qiskit_machine_learning/kernels/quantum_kernel.py

```diff
@@ -72,6 +74,21 @@ def __init__(
             training_parameters: Iterable containing ``Parameter`` objects which correspond to
                  quantum gates on the feature map circuit which may be tuned. If users intend to
                  tune feature map parameters to find optimal values, this field should be set.
+            evaluate_duplicates: Defines a strategy how kernel matrix elements are evaluated if
+                identical samples are found. Possible values are:
+
+                    - ``all`` means that all kernel matrix elements are evaluated, even the diagonal
+                        ones when training. This may introduce additional noise in the matrix.
+                    - ``non_diagonal`` when training the matrix diagonal is set to `1`, the rest
```

Comment:
I was wondering whether you had considered `off_diagonal` - or maybe thats not quite what you wanted. Since it evaluates everything off the diagonal it seemed to fit for me.

Reply:
Ah, you are totally right! Replaced with `off_diagonal`